ci/benchmark_parser.py

```diff
@@ -177,7 +200,12 @@ def check_mandatory_args(input_args):
     raw_results = pathlib.Path(args.results)
     if not args.key_sizes:
         print("Parsing benchmark results... ")
-        results = recursive_parse(raw_results, args.walk_subdirs, args.name_suffix)
+        hardware_cost = None
+        if args.ops_per_dollar:
+            ec2_costs = json.loads(pathlib.Path("ci/ec2_products_cost.json").read_text())
```

Comment:
no API call to get this information somewhere ? (storing it in a file is fine to avoid API failures, but we'll need to keep it up to date)

Reply:
Indeed I could do an API call for that but as you said It would require handling of network issues. Moreover, it introduce the need to use a third-party library like boto3 to perform such call, which is not installed on the machine.
IMHO, we should stay with a hardcoded list as long it doesn't grow (too much), prices won't change drastically anyway.